pkg/smbios/info.go

```diff
@@ -75,31 +84,31 @@ func (i *Info) GetTablesByType(tt TableType) []*Table {
 	return res
 }
 
-// GetBIOSInformation returns the Bios Information (type 0) table, if present.
-func (i *Info) GetBIOSInformation() (*BIOSInformation, error) {
-	bt := i.GetTablesByType(TableTypeBIOSInformation)
+// GetBIOSInfo returns the Bios Info (type 0) table, if present.
```

Comment:
Shouldn't you remove all the Get* as per go convention?

Reply:
I thought about it. I decided against it for now, but will perhaps do it later. There were a few places where removing the Get looked weird.